app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -113,6 +113,12 @@ def form_params_create
       # Get the storage manager.
       storage_manager_id = params[:storage_manager_id] if params[:storage_manager_id]
       options[:ems] = find_record_with_rbac(ExtManagementSystem, storage_manager_id)
+    when "ManageIQ::Providers::StorageManager::SwiftManager"
```

Comment:
This has to be `"ManageIQ::Providers::Openstack::StorageManager::SwiftManager"` now